src/components/LHNOptionsList/OptionRowLHN.js

```diff
@@ -107,12 +109,26 @@ function OptionRowLHN(props) {
             ReportUtils.isWaitingForIOUActionFromCurrentUser(optionItem) ||
             (optionItem.isTaskReport && optionItem.isTaskAssignee && !optionItem.isCompletedTaskReport && !optionItem.isArchivedRoom));
 
+    useFocusEffect(
+        useCallback(() => {
+            isFocusedRef.current = true;
+            return () => {
+                if (!props.isSmallScreenWidth) {
```

Comment:
Instead of HOC, let's use `useWindowDimensions` hook to get `isSmallScreenWidth`.

Reply:
Done.